shared/domains/chelonia/db.js

```diff
@@ -51,9 +56,12 @@ sbp('sbp/selectors/unsafe', ['chelonia/db/get', 'chelonia/db/set', 'chelonia/db/
 const dbPrimitiveSelectors = process.env.LIGHTWEIGHT_CLIENT === 'true'
   ? {
       'chelonia/db/get': function (key: string): Promise<Buffer | string | void> {
-        const id = sbp('chelonia/db/contractIdFromLogHEAD', key)
-        const value = (id ? sbp(this.config.stateSelector).contracts[id]?.HEAD : undefined)
-        return Promise.resolve(value)
+        if (isLogHead(key)) {
+          const id = getContractIdFromLogHead(key)
+          const value = sbp(this.config.stateSelector).contracts[id]?.HEAD
+          return Promise.resolve(value)
+        }
+        return Promise.resolve()
```

Comment:
Could you preserve the logic as it was before? (e.g. remove `isLogHead` and update `getContractIdFromLogHead`)?